scripts/settings.yaml

```diff
@@ -0,0 +1,511 @@
+---
+- group: base_station_mode
+  name: enable
+  type: boolean
+  units:
+  default value: 'false'
+  enumerated possible values: true,false
+  Description: Operate in base station mode
+  Notes: 'This flag ultimately determines whether the SBP message with identifier
+    MSG_BASE_POS will be calculated and sent.  Logically, setting this attribute to
+    "true" sets the Local Piksi as a base station and configures the unit to send
+    its surveyed location coordinates to the other Piksi(s) with which the base
+    station is communicating.  If "true", the Piksi that receives the base station''s
+    surveyed coordinates will also calculate and communicate its own pseudo absolute
+    RTK position based upon the received position of the Base Station.  '
+- group: base_station_mode
+  name: surveyed lat
+  type: Double
+  units: degrees
+  default value: '0'
+  enumerated possible values:
+  Description: Surveyed latitude of the Piksi connected to the console over usb
```

Comment:
"connected to the console over usb" - this may not be true, the Piksi could be in any configuration, some people want this documentation specifically so they can bypass the console :)